sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -146,6 +147,15 @@ internal class ViewAnnotationManagerImpl(
     return renderThread.viewAnnotationMode
   }
 
+  override val annotations: Map<View, ViewAnnotationOptions>
+    get() = HashMap<View, ViewAnnotationOptions>().apply {
+      idLookupMap.forEach { lookupMap ->
+        getViewAnnotationOptionsByView(lookupMap.key)?.let { viewAnnotationOptions ->
+          put(lookupMap.key, viewAnnotationOptions)
+        }
+      }
+    }
```

Comment:
Thought that you can easily use `map` kotlin function but apparently it maps to List for some reason. However, another notation is possible : 
```
  val annotations: Map<View, ViewAnnotationOptions>
    get() = idLookupMap.map { (view, _) ->
      view to getViewAnnotationOptionsByView(view)!!
    }.toMap()
```
with additional `toMap` that maps list of pairs to Map

Another q is - how to treat nullable `getViewAnnotationOptionsByView`? Seems it shouldn't happen, so probably worth to log an error or even use `!!`, wdyt?

Reply:
> Thought that you can easily use map kotlin function but apparently it maps to List for some reason

Yeah i did try that earlier but didn't want to use `!!` to be safe, so went ahead with currentl impl, also imo we should not add the entry if `getViewAnnotationOptionsByView ` is null (in case).

Reply:
Should we log an error though for the null case?